client/X11/generate_argument_docbook.c

```diff
@@ -42,8 +42,10 @@ LPSTR tr_esc_str(LPCSTR arg, bool format)
 					exit(-3);
 				}
 				if (format)
+					// coverity[buffer_size]
```

Comment:
Please no C++ comments

Reply:
`/* coverity[buffer_size] */` will be ok?

Reply:
yes, please. (sorry, but old C compilers complain horribly about these)